modules/ossm-configuring-jaeger.adoc

```diff
@@ -31,101 +30,148 @@ The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as
 
 |tracing
    enabled
-|This enables or disables tracing in {ProductShortName}. Jaeger is installed by default. If you do not want to install Jaeger, change the `enabled` value to `false`.
+|This parameter enables/disables tracing in {ProductShortName}. Jaeger is installed by default.
 |`true`/`false`
 |`true`
 
-|ingress
-   enabled
-|This enables/disables ingress.
-|`true`/`false`
-|`true`
+|jaeger
+   template
+|This parameter specifies which Jaeger deployment strategy to use.
+|* `all-in-one`- For development, testing, demonstrations, and proof of concept.
+* `production` - For production use.
+|`all-in-one`
 |===
 
-[id="ossm-configuring-jaeger-elasticsearch_{context}"]
-== Configuring Elasticsearch
-
-Jaeger can be configured for different storage backends:
-
-* *Memory* - Simple in-memory storage, only recommended for development, demo, or testing purposes. This is the default option for the `AllInOne` deployment strategy. Do NOT use for production environments.
-* *Elasticsearch* - For production use. This is the default option for the `Production` deployment strategy.
-
 [NOTE]
 ====
-The default template strategy in the `ServiceMeshControlPlane` resource is `AllInOne`. For production, the only supported storage option is Elasticsearch, therefore you must configure the `ServiceMeshControlPlane` to request the `production-elasticsearch` template strategy when you deploy {ProductShortName} within a production environment.
+The default template in the `ServiceMeshControlPlane` resource is the `AllInOne` deployment strategy which uses in-memory storage. For production, the only supported storage option is Elasticsearch, therefore you must configure the `ServiceMeshControlPlane` to request the `production` template when you deploy {ProductShortName} within a production environment.
```

Comment:
Is this `AllInOne` the same as `all-in-one` from earlier? Is so, should they be represented the same?